### ISSUE TITLE
Alerts cut out when VO highlight changes

### ISSUE THREAD

**Issue by zepumph** [phetsims/gravity-force-lab#196]
From https://github.com/phetsims/gravity-force-lab/issues/140#issuecomment-552534326, @terracoda found that position and mass alerts (mainly and consistently position alerts) would cut off because the VO highlight would be delayed in how it updates, and when it moved, the alert would be half way read through, and its movement would cause the alert to stop reading. Perhaps this has something to do with https://github.com/phetsims/gravity-force-lab/issues/194, but I'm not totally sure.

**Comment by jessegreenberg** [comment]
Someone with a mac will need to investigate this, I can look into this, potentially pairing with @zepumph if we have time.

**Comment by zepumph** [comment]
We didn't have time! Unassigning, but let me know @jessegreenberg if you would like assistance. Thanks for the help.

**Comment by jessegreenberg** [comment]
I tested for about 5 minutes on a macOS Safari with VoiceOver and didn't see this happen. I did notice that the VoiceOver highlight doesn't update with the DOM element, and I verified that the DOM element is getting positioned correctly:
<img width="437" alt="Screen Shot 2019-12-03 at 6 30 44 PM" src="https://user-images.githubusercontent.com/6396244/70099365-a1e06100-15fc-11ea-8a79-611c70990af7.png">

See the blue selection as safari indicates where the DOM element is, and the black VoiceOver highlight around where it was on focus.

Ill try on iOS VoiceOver next, maybe the issue is there.

**Comment by jessegreenberg** [comment]
I am getting alert cut-out on iOS VoiceOver.

**Comment by terracoda** [comment]
@jessegreenberg, do you think it would make a difference if we used `aria-live="assertive"`.

**Comment by jessegreenberg** [comment]
Yes that is definitely possible. But before changing that I would like to see if we can understand why this is happening for the sliders in this sim and not the sliders in other sims.

**Comment by terracoda** [comment]
@jessegreenberg and @zepumph, I just noticed in the PhET App for GFL Basics no interruptions happen.

The responses are not getting cut off for GFLB even though the VoiceOver's cursor takes a bit of time to catch up to the changed position of the moved mass sphere. Maybe the cut-off is not related to the cursor's delay?

**Comment by jessegreenberg** [comment]
I am no longer getting this behavior and I am wondering if this was fixed by https://github.com/phetsims/utterance-queue/commit/0f047918029132b457fd4ab95d419cb4441cdc26

**Comment by jessegreenberg** [comment]
I reverted the change and the issue returned pretty consistently. So I think this has been fixed. Ill make a new version for @terracoda to verify.

**Comment by jessegreenberg** [comment]
@terracoda can you please verify that this is fixed in https://phet-dev.colorado.edu/html/gravity-force-lab/2.2.0-dev.22/phet/gravity-force-lab_en_phet.html?a11y&supportsSound&supportsEnhancedSound?

**Comment by terracoda** [comment]
Responses are no longer being cut off!

**Comment by terracoda** [comment]
@jessegreenberg, it sounds really good. I'll close this issue.
